issuer/views.py

```diff
@@ -210,9 +218,18 @@ def add_issuance(self, issuance):
         )
         return issuance
 
+class IssueResponse(HttpResponse):
+    def __init__(self, data, callback, **kwargs):
+        super().__init__(data, **kwargs)
+        self.callback = callback
 
-class UnsignedCertificatesView(View):
+    def close(self):
+        super().close()
+        self.callback()
+
+class IssueCertificatesView(View):
     def post(self, request):
+        unsigned_certs_batch = []
         for person_issuance in PersonIssuances.objects.filter(is_issued=False, is_approved=True).exclude(person__public_address=''):
             issuance = Issuance.objects.get(id=person_issuance.issuance.id)
```

Comment:
I know this wasn't added here but can we clean it up, it's making an unnecessary db call and can be replaced with `issuance = person_issuance.issuance`